DEVELOPERS.md

```diff
@@ -53,6 +53,12 @@ Open up a REPL:
 $ cabal exec -- ghci Model.hs
 ```
 
+## Sass
+
+- In a separate terminal, run `sass --watch sass:static/css`
```

Comment:
@tysongach @smharley would that be valuable?

I would just do `^Rsass<enter>`, myself.